src/microbit/model/microbit_model.py

```diff
@@ -9,6 +10,7 @@ def __init__(self):
         self.button_a = Button()
         self.button_b = Button()
         self.__start_time = time.time()
+        self.display = Display()
 
     def sleep(self, n):
         time.sleep(n / 1000)

```

Comment:
Oh you have a sleep function here

Reply:
dont want to use this one because of circular imports though, so i added a static function to display